core/os/file_access.cpp

```diff
@@ -252,10 +252,10 @@ String FileAccess::get_token() const {
 
 class CharBuffer {
 	Vector<char> vector;
-	char stack_buffer[256];
+	char stack_buffer[256] = { 0 };
```

Comment:
Is there any significant performance impact to doing this, as opposed to using this buffer knowing that it contains garbage data initially?

Reply:
Since there is no way to unsafely use this array, it is really better to not initialize this data.
This class is only used in one place - `get_line` - function.